src/modules/roc_sndio/target_sox/roc_sndio/sox_backend.cpp

```diff
@@ -196,5 +196,36 @@ ISource* SoxBackend::open_source(core::IAllocator& allocator,
     return source.release();
 }
 
+void SoxBackend::get_drivers(core::Array<DriverInfo>& arr) {
+    const sox_format_tab_t* formats = sox_get_format_fns();
+    char const* const* format_names;
+    size_t n;
+    for (n = 0; formats[n].fn; n++) {
+        sox_format_handler_t const* handler = formats[n].fn();
+        if (!(handler->flags & SOX_FILE_DEVICE)
+            || ((handler->flags & SOX_FILE_DEVICE)
+                && !(handler->flags & SOX_FILE_PHONY))) {
+            for (format_names = handler->names; *format_names; ++format_names) {
+                if (!strchr(*format_names, '/')) {
```

Comment:
Could you explain why do we need this check?

Reply:
So I've ran it with and without this check and it makes no difference.  The only reason I put this in is because the sox.c library also includes it, (display_supported_formats I used this function as a guideline to extract the names from sox_format_fns).  I'm not sure why they use it though, none of the names have '/', but I felt that if they included it I should do the same.

Reply:
OK, I see, let's keep it :) Could you then add a comment that we replicate the behavior of the display_supported_formats() from sox?